lib/data_filter/filter_set.rb

```diff
@@ -92,9 +92,9 @@ def create(&block)
       #   to filter
       # @option opts [Object] :by the value we want to use when
       #   filtering the data item
-      def like_filter(field_sym, opts = {})
+      def like_filter(field_sym, opts = {}, normalize_regex = /[^\w\s]/)
```

Comment:
What about having this be another option instead? That way it'd look something like:

```rb
like_filter by: :name, normalize_regex: /[^\w\s^.^@^-]/
```

Also could you add to the doc block above as well as to the examples in `README.md`?